lib/replicache.dart

```diff
@@ -195,9 +195,7 @@ class Replicache implements ReadTransaction {
     _opened = _staticInvoke(_name, 'open', repmInvoke: _repmInvoke);
     _root = _getRoot();
     await _root;
-    if (_syncInterval != null) {
-      await sync();
-    }
+    await sync();
```

Comment:
[fix?] Doesn't this breaks the test? It is important that a sync does not happen when testing.

Reply:
Yes, it does, thanks for debugging for me :).